code/game/machinery/autolathe.dm

```diff
@@ -206,7 +206,7 @@
 				busy = TRUE
 				use_power(power)
 				icon_state = "autolathe_n"
-				var/time = is_stack ? 32 : 32*coeff*multiplier
+				var/time = is_stack ? 10 : 10 * coeff * multiplier
```

Comment:
Hey, care to make a variable for this and set it to 20 for the public lathe, so I won't have to?
I don't believe simple freeloaders should benefit from reduced printing times this much.

Reply:
sure